collector/prometheus/src/main/java/org/apache/karaf/decanter/collector/prometheus/PrometheusCollector.java

```diff
@@ -71,28 +73,39 @@ public void activate(Dictionary<String, Object> properties) throws Exception {
     public void run() {
         try {
             URLConnection connection = prometheusURL.openConnection();
-            Map<String, Object> data = new HashMap<>();
-            data.put("type", "prometheus");
+            String topic = (properties.get(EventConstants.EVENT_TOPIC) != null) ? (String) properties.get(EventConstants.EVENT_TOPIC) : "decanter/collect/prometheus";
```

Comment:
Just a reminder:  it's possible to override via configuration, in `etc/org.apache.karaf.decanter.collector.prometheus.cfg`:

```
event.topics=/foo/bar
```

It's documented here https://karaf.apache.org/manual/decanter/latest-2/html/#_alerters in the "Using existing appenders".